src/Campaigns/ListTable/Columns/StatusColumn.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Give\Campaigns\ListTable\Columns;
+
+use Give\Campaigns\Models\Campaign;
+use Give\Framework\ListTable\ModelColumn;
+
+/**
+ * @unreleased
+ */
+class StatusColumn extends ModelColumn
+{
+    /**
+     * @unreleased
+     */
+    public static function getId(): string
+    {
+        return 'status';
+    }
+
+    /**
+     * @unreleased
+     */
+    public function getLabel(): string
+    {
+        return __('Status', 'give');
+    }
+
+    /**
+     * @unreleased
+     *
+     * @param Campaign $model
+     */
+    public function getCellValue($model)
+    {
+        return $model->status->getValue();
```

Comment:
We should return a translatable string here, not the status value. 
Something like:
```php
switch($model->status->getValue()){
     case 'active':
          return __('Active', 'give');
     ...
}
```

Reply:
Resolved in 3e1af62cccdf32ef86d02f13e54f19d9a599598f